week2-section1/exercise2/Main.cpp

```diff
@@ -0,0 +1,65 @@
+/*
+Exercise 2 - Military Clock
+Create a class called MilitaryClock that stores and displays the military time.
+*/
+
+#include <iostream>
+
+
+class MilitaryClock
+{
+
+private:
+
+	short hours;
```

Comment:
Remember, we are using PascalCase. (I know the example of the exercise don't, I'll update them, but please follow convention in spite of the exercises examples).